packages/web-example/src/local.ts

```diff
@@ -123,15 +119,15 @@ const runTest = async () => {
   }
 
   console.log(sockets);
-  const client = new Client(session, clientIndex, partyIndexes, endpoints, sockets, share, pubKey, true, tssImportUrl);
+  const client = new Client(session, clientIndex, partyIndexes, endpoints, sockets, share, pubKey, true, tssLib);
```

Comment:
This is a breaking change.

It is good however because it then uses the lib packaged with it instead of also downloading it.

We possibly could get it down to this:
```
const client = new Client(session, clientIndex, partyIndexes, endpoints, sockets, share, pubKey, true);
```

Then we would not have to publish tss-lib separately again.

Reply:
See here: https://github.com/torusresearch/tss-client/pull/45/files#diff-d07d17634456411d84465fecb7e133cd7705c49bd30e62541156b41ec8d3b23fL169

Reply:
I've extended the PR description above a bit to make it more clear what this PR aims at and what the context is. Broadly speaking, we are working on the integration of both clients libs in MPC Core Kit.

- We want to keep the WASM package separate from the client lib, so that we can optionally include it and maintain a small package size of core kit, if one of the libs is not used.
- Yes, this here is a breaking change. We will need to bump the major version for that. How it works: The client gets the lib in the constructor, instead of passing the lib at each function call. Previously, a URL to the WASM was also passed in the constructor, but only used for the web worker. This PR proposes to remove web worker functionality completely, as it was not used and in fact is currently not functional because of a defect in the Rust code (see [dkls](https://github.com/torusresearch/dkls/commit/1a3cb148561a695822d80d4f6f487c954972cffd)).

Reply:
Ok, then function can be removed entirely now.

Reply:
Have submitted a PR to dkls for this.